src/main/resources/validationrules/01_validate_request.drl

```diff
@@ -33,17 +32,29 @@ $request        : Request(requestType == RequestType.CREATE)
 
 then
 $requestedRole.setStatus(Status.APPROVED);
-$request.setLog(String.format("Requested has been approved by rule : R11_role_validation "));
-$requestedRole.setLog(String.format("Requested Role has been approved by rule : R11_role_validation "));
+$request.setLog(String.format("Requested Role has been approved by rule : R01_role_validation_for_organization_pattern "));
+$requestedRole.setLog(String.format("Requested Role has been approved by rule : R01_role_validation_for_organization_pattern "));
```

Comment:
it is 2 times

Reply:
These are two different varibales,one is request and other is requested Role.